src/router/index.js

```diff
@@ -25,9 +25,12 @@ const router = createRouter({
       component: Dashboards,
     },
     {
-      path: '/dashboards/dashboard',
-      name: 'dashboard',
+      path: '/dashboards/:id',
       component: Dashboard,
+      name: 'dashboard',
+      props: (route) => ({
+        dashboardId: route.params.dashboardId,
```

Comment:
Since path is expecting `:id`, I imagine `route.params.dashboardId` wouldn't work as desired. Has this code been tested?

Reply:
I tested it and it's working. I just changed the name to make more sense.